.gitignore

```diff
@@ -35,6 +35,8 @@ graphql.config.json
 public/status.json
 .env
 .env.*local
+env.test
+env.*local
```

Comment:
```suggestion
```

Haven't looked properly yet but can we get rid of these from the `.gitignore`? :-)